shell/Dockerfile

```diff
@@ -26,8 +26,26 @@ RUN tdnf install -y \
 	tar \
 	file \
 	bpftool \
+	ig \
 	&& tdnf clean all
 
+# Create the entrypoint script to mount debugfs
+SHELL ["/bin/bash", "-c"]
+
+RUN echo $'#!/bin/bash\n\
+if ! mountpoint -q /sys/kernel/debug; then\n\
+mount -t debugfs none /sys/kernel/debug\n\
```

Comment:
[For reference only] This will only persist for the lifetime of the container and will only mount debugfs inside the container’s namespace (not accessible to other containers). The host will not see the mount; only processes inside the container (and potentially other containers in the same Pod with shared namespaces) will have access